app/client/src/widgets/InputWidgetV2/widget/index.tsx

```
@@ -18,7 +18,7 @@ import {
   INPUT_TEXT_MAX_CHAR_ERROR,
 } from "ee/constants/messages";
 import type { SetterConfig, Stylesheet } from "entities/AppTheming";
-import { debounce, isNil, isNumber, merge, toString } from "lodash";
+import { isNil, isNumber, merge, toString } from "lodash";
```

Biome (1.9.4)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)